priv/companies/keila.exs

```diff
@@ -0,0 +1,14 @@
+%{
+  industry: "Marketing/Sales",
```

Comment:
@wmnnd can you update this to align the industry with the set of industries that we support? You can find a list of supported industries in https://github.com/beam-community/elixir-companies/blob/main/lib/companies/industries.ex#L9-L23